tools/cli/adminDBCleanCommand.go

```diff
@@ -66,8 +67,14 @@ func AdminDBDataDecodeThrift(c *cli.Context) {
 					ErrorAndExit("cannot encode back to confirm", err)
 				}
 				if bytes.Equal(data, data2) {
-					fmt.Printf("=======Decode into type %v ========\n", typeName)
-					fmt.Println(anyToString(t, true, 0))
+					fmt.Printf("======= Decode into type %v ========\n", typeName)
```

Comment:
do we... want these if we're working with JSON?

Reply:
frequently no, I think.
but json doesn't contain type information, so both _are_ useful.  it'd probably be better to have a `--format` flag here though.